addOns/websocket/src/main/zapHomeFiles/scripts/templates/websocketpassive/Passive WebSocket Scan Template.js

```diff
@@ -0,0 +1,87 @@
+// Passive scan rules should not send messages
+// Right click the script in the Scripts tree and select "enable"  or "disable"
+
+OPCODE_CONTINUATION = 0x0;
+
+OPCODE_TEXT = 0x1;
+OPCODE_BINARY = 0x2;
+
+OPCODE_CLOSE = 0x8;
+OPCODE_PING = 0x9;
+OPCODE_PONG = 0xA;
+
+RISK_INFO 	= 0;
+RISK_LOW 	= 1;
+RISK_MEDIUM = 2;
+RISK_HIGH 	= 3;
+
+CONFIDENCE_LOW = 1;
+CONFIDENCE_MEDIUM = 2;
+CONFIDENCE_HIGH = 3;
+
+/**
+ * This function scans passively WebSocket messages. The scan function will be called for
+ * messages via ZAP.
+ *
+ * @param helper - the WebSocketPassiveHelper interface provides the raiseAlert method in order
+ *               to raise the appropriate alerts
+ * Some useful function(s) about  WebSocketPassiveHelper:
+ * helper.getAlertRaiser() -> Returns an WebSocketAlertRaiser instance which is used
+ *                            for building and raising alerts.
+
+ * * Some useful functions about WebSocketAlertRaiser:
+ * * alertRaiser.setRiskConfidence(risk, confidence) -> Sets the Risk and the Confidence of the alert. (by default RISK_INFO, CONFIDENCE_MEDIUM).
+ * * alertRaiser.setName(name)                       -> Sets the name (by default "").
+ * * alertRaisersetDescription(description)          -> Sets a description about potential threat (by default "").
+ * * alertRaiser.setParam(param)                     -> Sets in which parameter the threat noticed (by default "").
+ * * alertRaiser.setAttack(attack)                   -> Sets the possible attack (by default "").
```

Comment:
We should not show this method in passive scripts, passive alerts should not have an attack.